docs/building-modules/module-interfaces.md

```diff
@@ -103,7 +103,7 @@ Request types, which define structured interactions from users, must be defined
 
 Here is an example of a request to buy a name from the `nameservice` module:
 
-+++ https://github.com/cosmos/sdk-tutorials/blob/master/nameservice/x/nameservice/client/rest/tx.go#L14-L19
++++ https://github.com/cosmos/cosmos-sdk/blob/7f59723d889b69ca19966167f0b3a7fec7a39e53/x/bank/client/rest/tx.go#L15-L19
```

Comment:
I approve of changing theblinks, using either a tag or commit hash. But could you use the same commit hash/tag in all links?

Reply:
Understood, I tried it with the same commit hash that is being used by other updated URLs but it's invalid.

Same commit hash: https://github.com/cosmos/sdk-tutorials/blob/86a27321cf89cc637581762e953d0c07f8c78ece/nameservice/x/nameservice/internal/keeper/querier.go#L19-L32

`86a27321cf89cc637581762e953d0c07f8c78ece`

Invalid URL with the same commit hash: https://github.com/cosmos/cosmos-sdk/blob/86a27321cf89cc637581762e953d0c07f8c78ece/x/bank/client/rest/tx.go#L15-L19

FYI: I've updated the broken links with the same links from `master` so they should be fine.